pkg/plugins/handler.go

```diff
@@ -68,7 +67,7 @@ func (pl Handler) LoadAMQPPlugin(wg *sync.WaitGroup, scConfig *common.SCConfigur
 }
 
 // LoadPTPPlugin loads ptp plugin
-func (pl Handler) LoadPTPPlugin(wg *sync.WaitGroup, scConfig *common.SCConfiguration, fn func(e ceevent.Event) error) error {
+func (pl Handler) LoadPTPPlugin(wg *sync.WaitGroup, scConfig *common.SCConfiguration, fn func(e interface{}) error) error {
```

Comment:
Since the PTP plugin only takes a cloud-native events, it should not take any other type.

Reply:
I got this build failure if I use ceevent.Event
```
plugins/ptp_operator/ptp_operator_plugin.go:75:62: cannot use fn (type func("github.com/redhat-cne/sdk-go/pkg/event".Event) error) as type func(interface {}) error in argument to "github.com/redhat-cne/sdk-go/v1/amqp".CreateNewStatusListener
make: *** [Makefile:59: build-plugins] Error 2
```